announcement.md

```diff
@@ -2,7 +2,7 @@
 
 After a successful [inaugural session](inauguralsession.md) we aim to bring VVirtual Science Forum closer to its conceived goal: facilitate scientific communities organizing virtual events.
```

Comment:
After a successful [inaugural session](inauguralsession.md) we aim to bring the Virtual Science Forum closer to its conceived goal: aid researchers to organizing virtual events.

Reply:
Change has been addressed in latest commit